src/components/ha-clickable-list-item.ts

```diff
@@ -0,0 +1,48 @@
+import { ListItem } from "@material/mwc-list/mwc-list-item";
+import { css, CSSResult, customElement } from "lit-element";
+import { html } from "lit-html";
+
+@customElement("ha-clickable-list-item")
+export class HaClickableListItem extends ListItem {
```

Comment:
mwc-list-item doesn't like being wrapped inside anchor tags:
https://github.com/material-components/material-components-web-components/issues/1095#issuecomment-715927210

We have to extend ListItem to retain keyboard navigation behavior